Helper/Order.php

```diff
@@ -216,30 +218,42 @@ class Order extends AbstractHelper
      */
     protected $customerCreditCardCollectionFactory;
 
+    /** @var CreditmemoFactory */
+    protected $creditmemoFactory;
+
+    /** @var CreditmemoManagementInterface  */
+    protected $creditmemoManagement;
+
     /**
+     * Order constructor.
      * @param Context $context
-     * @param ApiHelper $apiHelper
+     * @param Api $apiHelper
```

Comment:
As i see above in use
use Bolt\Boltpay\Helper\Api as ApiHelper;
For what reason it was changed to Api class?

But in `__constructor` still placed `ApiHelper`
```
public function __construct(
        Context $context,
        ApiHelper $apiHelper,
```

Reply:
Updated